packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,75 @@
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop          | Type                                                       | Default     | Description                                                                                                                                                                                       | Required |
+| ------------- | ---------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| value         | `number`                                                   | `undefined` | The value to be rendered within the component                                                                                                                                                     | ✅       |
+| variant       | `positive`, `negative`, `notice`, `information`, `blue`    | `undefined` | The variant of the amount to be rendered                                                                                                                                                          |          |
+| fontWeight    | `bold`, `regular`                                          | `bold`      | The weight of the amount to be rendered                                                                                                                                                           |          |
+| suffix        | `none`, `decimals` , `humanize`                            | `decimals`  | The type of affix to be displayed with the currency value, (none, decimals, and humanize) that respectively indicate no affix, the number of decimal places, and the scale of the currency value. |          |
+| size          | `3xlarge`, `2xlarge`, `xlarge`, `large`, `medium`, `small` | `3xlarge`   | The size of the amount to be rendered                                                                                                                                                             |          |
+| isAffixSubtle | `true`, `false`                                            | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small and faded                                                                                                          |          |
+
+### Sample Usage
+
+```jsx
+import { Amount } from '@razorpay/components';
+
+<Amount
+  suffix="decimals"
+  size="2xlarge"
+  isAffixSubtle={true}
+  fontWeight="regular"
+  variant="information"
+  value={1000}
+/>;
+```
+
+## Open Questions
+
+- Should we call `isSuffixPrefixHighlighted` or `isHighlighted` or `isAffixHighligted`, considering when this value is true the value will be highligted and not the affix.
+
+#### Dev
```

Comment:
@Priyanshu-C lets add all the answers to the questions based on the discussions inline so we can then focus on closing the remaining ones?